data_tools/buildhub/utils.py

```diff
@@ -0,0 +1,22 @@
+from os.path import basename
+
+
+def build_record_id(url):
+    # Generate an ID that looks like that: firefox_53.0b9_linux-i686_en-US
+    parts = basename(url).split('.')
+    url_parts = url.split('/')
+
+    product_name, major_version = parts[0].split('-')
```

Comment:
Maybe we could pass an archive record instead of parsing this again. (plus, `mobile` != `fennec`)